options/rtdl/generic/linker.cpp

```diff
@@ -885,6 +899,12 @@ void Loader::submitObject(SharedObject *object) {
 	if(_linkSet.get(object))
 		return;
 
+	// at this point the object is loaded and we can fill in its debug struct
+	// the linked list fields will be filled later
```

Comment:
Comments should be proper sentences (capitalization + punctuation, also elsewhere).